src/main/java/thePackmaster/cards/pinnaclepack/MeatyCroquettesSpecialColourless.java

```diff
@@ -0,0 +1,31 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class MeatyCroquettesSpecialColourless extends AbstractPinnacleCard {
+
+    public final static String ID = makeID("MeatyCroquettesSpecialColourless");
+    private static final int MAGIC = 3;
+    private static final int UPGRADE_MAGIC = 3;
+
+
+    public MeatyCroquettesSpecialColourless() {
```

Comment:
*Meaty Croquette*
This is fine for a token card, but it shouldn't be a power that exhausts (powers don't exhaust).

Making this a skill like the other Croquettes would increase the similarity between the Croquettes, which helps reduce their complexity, and would fix this issue.